sdk/python/feast/infra/key_encoding_utils.py

```diff
@@ -62,3 +62,27 @@ def serialize_entity_key(entity_key: EntityKeyProto) -> bytes:
         output.append(val_bytes)
 
     return b"".join(output)
+
+
+def deserialize_entity_values(entity_values: ByteString) -> List[Any]:
+    deserialized_entity_values = []
```

Comment:
might be worth adding a comment that this is mostly partially reversing serialize_entity_key above, but assumes the input has already stripped out the entity keys themselves? Kind of hacky, but seems fine for now.

Reply:
hey good point